__tests__/serverSpec.jsx

```diff
@@ -0,0 +1,40 @@
+
+import { getDummyTitle, getDummyMaps } from '../client/src/http-helpers';
+
+describe('/map route tests', () => {
+  test('should not have an error for a get request to /map', () => {
+    getDummyMaps((err, result) => {
+      expect(result).toHaveLength(200);
+    });
+  });
+  test('should receive 200 status for a get request to /map', () => {
+    getDummyMaps((err, result) => {
+      expect(result.getResponseHeader()).toBe(true);
```

Comment:
just checking if it's sending back a header which is what it would do even with an error maybe specify and check for the actual 200 code